vm-builder/files/inittab

```diff
@@ -1,5 +1,7 @@
 ::sysinit:/neonvm/bin/vminit
 ::once:/neonvm/bin/touch /neonvm/vmstart.allowed
+# we might be using hvc0 console, so we need to print the kernel version in user space, after virtio console is initialized
+::once:/neonvm/bin/uname -a
```

Comment:
For comparison:
```
root:~# cat /proc/version
Linux version 6.12.24 (root@buildkitsandbox) (x86_64-linux-gnu-gcc (Ubuntu 13.3.0-6ubuntu2~24.04) 13.3.0, GNU ld (GNU Binutils for Ubuntu) 2.42) #1 SMP PREEMPT_DYNAMIC Wed Apr 23 17:31:20 UTC 2025

root:~# uname -a
Linux vm-compute-msf-unreal-xxdpm 6.12.24 #1 SMP PREEMPT_DYNAMIC Wed Apr 23 17:31:20 UTC 2025 x86_64 GNU/Linux
```

Not a blocker, but I slightly prefer the first option.

Reply:
Changed in [79041126e09d871f68da63694055f530891d906a](https://github.com/neondatabase/autoscaling/pull/1366/commits/79041126e09d871f68da63694055f530891d906a)

Reply:
```
[    0.304055] EXT4-fs (vda): resizing filesystem from 262144 to 2097152 blocks
[    0.307157] EXT4-fs (vda): resized filesystem to 2097152
The filesystem on /dev/vda is now 2097152 (4k) blocks long.

Linux version 6.6.64 (root@buildkitsandbox) (x86_64-linux-gnu-gcc (Ubuntu 13.3.0-6ubuntu2~24.04) 13.3.0, GNU ld (GNU Binutils for Ubuntu) 2.42) #1 SMP PREEMPT_DYNAMIC Mon Feb  3 10:05:38 UTC 2025
udhcpc: started, v1.35.0
starting up with netlink and the input layer
```